index.html

```diff
@@ -1748,7 +1752,11 @@ <h2>Roland Turner</h2>
                                         <div class="hover-state text-center preserve3d">
                                             <div class="social-links vertical-align">
                                                 <a href="http://justinlee.sg" target="default"><i class="icon fa fa-external-link"></i></a>
-                                                <a href="https://twitter.com/triplez82" target="default"><i class="icon social_twitter"></i></a>
+												<a href="https://twitter.com/triplez82" target="default"><i class="icon social_twitter"></i></a>
```

Comment:
You need to remove extra spaces , align them as earlier. Do it for other added lines too.